src/Swashbuckle.AspNetCore.ReDoc/Swashbuckle.AspNetCore.ReDoc.csproj

```diff
@@ -50,6 +54,11 @@
     <AdditionalFiles Include="PublicAPI\$(_TargetFrameworkIdentifier)\PublicAPI.Shipped.txt" />
     <AdditionalFiles Include="PublicAPI\$(_TargetFrameworkIdentifier)\PublicAPI.Unshipped.txt" />
   </ItemGroup>
+  
+  <ItemGroup>
+    <Compile Include="..\Swashbuckle.AspNetCore.SwaggerUI\CompressedEmbeddedFileResponser.cs" Link="CompressedEmbeddedFileResponser.cs" />
+    <Compile Include="..\Swashbuckle.AspNetCore.SwaggerUI\HttpContextAcceptEncodingCheckExtensions.cs" Link="HttpContextAcceptEncodingCheckExtensions.cs" />
```

Comment:
It would be better to put these into the [Shared](https://github.com/domaindrivendev/Swashbuckle.AspNetCore/tree/master/src/Shared) folder and consume as shared source in both projects.